workers/unity/Packages/com.improbable.gdk.tools/Common.cs

```diff
@@ -66,65 +66,32 @@ internal static string GetThisPackagePath()
         /// </summary>
         public static string GetPackagePath(string packageName)
         {
-            var manifest = ParseDependencies(ManifestPath);
-
-            if (!manifest.TryGetValue(packageName, out var path))
+            // Get package info
+            var request = Client.List(true);
+            while (!request.IsCompleted)
             {
-                throw new Exception($"The project manifest must reference '{packageName}'.");
+                // Wait for the request to complete
             }
 
-            if (!path.StartsWith("file:"))
-            {
-                throw new Exception($"The '{packageName}' package must exist on disk.");
-            }
+            var result = request.Result;
```

Comment:
Nit - can just be wrapped into the line below - there aren't any other usages of `result`